interface/cpp.cc

```diff
@@ -553,13 +553,42 @@ void cpp_generator::print_methods_impl(ostream &os, const isl_class &clazz)
 	}
 }
 
+/* Map isl function names to C++ operators.
+ */
+static const char *op_map[][2] = {
```

Comment:
Sven prefers to not use C++11 in the generator yet (but the generated code is C++11). Hence, I use now a std::map, instead.